R/tm_t_summary_by.R

```diff
@@ -124,7 +124,11 @@ template_summary_by <- function(parentname,
   parsed_basic_table_args <- teal.widgets::parse_basic_table_args(
     teal.widgets::resolve_basic_table_args(
       user_table = basic_table_args,
-      module_table = teal.widgets::basic_table_args(show_colcounts = TRUE, title = table_title)
+      module_table = teal.widgets::basic_table_args(
+        show_colcounts = TRUE,
+        title = table_title,
+        main_footer = "N represents the number of unique subject IDs such that the variable has NA values."
```

Comment:
I don't like the fact that we have in encoding and footer `NA` and then table shows "Missing", what do you think on being a bit more consistent? (and maybe reflect the same in encoding)

1.

```suggestion
        main_footer = "N represents the number of unique subject IDs such that the variable has NA/Missing values."
```

2.

```suggestion
        main_footer = "N represents the number of unique subject IDs such that the variable has missing values (NA)."
```

Reply:
Awesome! it has a linter error for long lines though :see_no_evil: